sql/bank_test.go

```diff
@@ -26,16 +26,36 @@ import (
 	"testing"
 
 	"github.com/cockroachdb/cockroach/server"
+	cockroachSql "github.com/cockroachdb/cockroach/sql"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 var maxTransfer = flag.Int("max-transfer", 999, "Maximum amount to transfer in one transaction.")
 var numAccounts = flag.Int("num-accounts", 999, "Number of accounts.")
 
-// BenchmarkBank mirrors the SQL performed by examples/sql_bank, but structured
-// as a benchmark for easier usage of the Go performance analysis tools like
-// pprof, memprof and trace.
-func BenchmarkBank(b *testing.B) {
+// BenchmarkBankNoDescriptorCache mirrors the SQL performed by examples/sql_bank,
+// but structured as a benchmark for easier usage of the Go performance analysis
+// tools like pprof, memprof and trace.
+// It does not make use of the system cache.
+func BenchmarkBankNoDescriptorCache(b *testing.B) {
```

Comment:
I'd prefer the flag. I think we'll dump this benchmark pretty quickly. And we certainly don't want to suggest that you run with descriptor caching disabled. I think you just want to keep this around because the performance difference is so dramatic.